exercisePPJ_06_I/FunctionPPJ_06_I.java

```diff
@@ -0,0 +1,52 @@
+public class FunctionPPJ_06_I {
+	
+	private double binaryWrt[] = {0,0,0,0,0,0,0,0};
+	private double wrtDouble = 0;
+	
+	public void setFunction(double wrtD) {
+		wrtDouble = wrtD;
+	}
+	
+	public void getFunction() {
+		
+		for(int counter = 7; counter >= 0; counter--) {
```

Comment:
counter has fixed value. It's not wrong, because bit has a fixed length, but just fr your information: we should avoid doing that (in most cases).
What would you use instead of that?

Reply:
@MatDls 
in some situation using infinitive loop (while) is not advisable, I think we should select the best way in specific problem. when i use while this program doing one operation all time. when i use for, program doing all operation 7 times.